### ISSUE TITLE
How to cut the neck in half?

### ISSUE THREAD

**Issue by mengcius** [Zielon/INSTA#29]
Your dataset cut the neck of the training set in half, but the test set is the whole neck. I wonder how you cut it?

![1](https://github.com/Zielon/INSTA/assets/46582759/605c626d-80ad-4d98-bf6c-79a7926a6c4b)
![3187](https://github.com/Zielon/INSTA/assets/46582759/81391875-059b-46ad-80d7-13dcb112da39)

**Comment by Zielon** [comment]
I used FLAME masks. You would need to rasterize a mesh using optimized camera parameters and combine it with segmentation masks. FLAME provides files with masks defined on the topology and their ID.
